.Gruntfile.babel.js

```diff
@@ -310,6 +326,7 @@ module.exports = (grunt) => {
           // https://github.com/vuejs/rollup-plugin-vue/blob/master/src/index.ts
           // https://github.com/vuejs/vue-component-compiler#api
           css: false,
+          needMap: false,
```

Comment:
Woah hold on there, this is a separate PR (#629) and so shouldn't be here.

Reply:
True, my bad, it was to help me during the development as it passed me by. removed.